source/guides/guides/continuous-integration.md

```diff
@@ -121,10 +121,16 @@ See our {% url 'examples' https://on.cypress.io/docker %} for additional informa
 
 If you are not using one of the above CI providers then make sure your system has these dependencies installed.
 
+## Ubuntu / Debian
 ```shell
 apt-get install xvfb libgtk2.0-0 libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2
 ```
 
+## RHEL / CentOS
+```shell
+yum install xorg-x11-server-Xvfb gtk2 libnotify GConf2 nss libXScrnSaver alsa-lib
```

Comment:
So I was looking at the docs, and I think instead of adding every dependency here in Markdown, without really testing it, we could just 
- keep existing list of Debian dependencies (since this is the most common image I believe)
- point at our https://github.com/cypress-io/cypress-docker-images repo where the dependencies are actually used to make and test images. In these docs we can say "for the up-to-date list see Docker image <link>"

If a library is needed to run Cypress, one can always open an issue in cypress-docker-images with a test case that fails to execute, and adding the library solves

Reply:
what about something like this?

```markdown
# Dependencies

If you are not using one of the above CI providers then make sure your system has the required dependencies installed.

An up-to-date list of required dependencies for supported platforms can be found in the `Dockerfile` for each platform in the [`cypress-io/cypress-docker-images`](https://github.com/cypress-io/cypress-docker-images) repository
```

I'm not sure I see the value in keeping only the debian list if we're going to suggest they check the `Dockerfile`s for an up-to-date list anyway, but if you feel strongly about it I can do that.